modules/GitterIRCBridge.js

```diff
@@ -0,0 +1,106 @@
+var GitterChat = require("./GitterChat");
+var IRCChat = require("./IRCChat");
+var octonode = require("octonode");
+
+function Bridge(bridge_config) {
+	var self = this;
+	
+	if (!bridge_config || typeof bridge_config !== "object") { throw new Error("A bridge config must be specified in form of an object"); }
+	
+	var github = octonode.client(bridge_config.github.auth);
+	
+	var gitter = new GitterChat(
+		bridge_config.gitter.token,
+		bridge_config.gitter.channel,
+		bridge_config.cmd_prefix
+	);
+	var irc = new IRCChat(
+		bridge_config.irc.host,
+		bridge_config.irc.port,
+		bridge_config.irc.nick,
+		bridge_config.irc.password,
+		bridge_config.irc.channel,
+		bridge_config.irc.welcome_message,
+		bridge_config.irc.ssl || false,
+		bridge_config.cmd_prefix
+	);
+	
+	gitter.on("message", function(from, msg) {
+		msg = msg.replace(new RegExp(bridge_config.github.org + "/" + bridge_config.github.repo + "#", "gi"), "https://github.com/" + bridge_config.github.org + "/" + bridge_config.github.repo + "/issues/");
```

Comment:
this should probably be separated into a few different functions where it makes sense.